### ISSUE TITLE
MinTime property wrongly calculate fc-not-start/end

### ISSUE THREAD

**Issue by ZadravecMarko** [fullcalendar/fullcalendar-scheduler#555]
When set the value `MinTime`, the class of the events are calculated wrongly.
For example, if MinTime is 6.00 then, all Events before 6 has class fc-not-start although whole Events is visible.

Example [here](https://jsfiddle.net/Ly9rsoc1/2/)

The 2 events on Friday 7/13 has < sign witch shouldn't be the case...

**Comment by acerix** [comment]
I can see it's not rendering correctly in version 3 but it seems to be fixed in version 4:

https://codepen.io/anon/pen/gNyzKd?&editable=true&editors=001

**Comment by ZadravecMarko** [comment]
No, even in your example, the arrow is still there even though the event is whole visible.

**Comment by acerix** [comment]
The `minTime` is 7am and the event starts before 7am, which means the beginning is hidden, and the end is hidden because it goes into the next day.   I added an event that doesn't go into the hidden time and it doesn't have the arrows:

https://codepen.io/anon/pen/gNyzKd?editors=0010

**Comment by ZadravecMarko** [comment]
Hi, ok I see now,... I see that if you set the minTime, it clip all events in one day. In this case it works as it was probably attended...
But, what if you would like to have "normal behaviour", that event extends normally over all days that are available, but that day starts with different time? (just shift of hours)?
